data/json/items/chemicals_and_resources.json

```diff
@@ -540,31 +540,73 @@
     "count" : 10
    },
    {
-    "type" : "AMMO",
+    "type" : "COMESTIBLE",
```

Comment:
This change is a bad thing: pure ethanol isn't really a comestible. Having it as comestible won't result in any meaningful addition to the game, but it will pollute the consumption menu and cause accidental consumptions.